doc/source/actors.rst

```diff
@@ -376,6 +376,9 @@ This allows you to retrieve the actor from any job in the Ray cluster.
 This can be useful if you cannot directly
 pass the actor handle to the task that needs it, or if you are trying to
 access an actor launched by another driver.
+Note that the actor will still be garbage-collected if no handles to it
+exist. See `Actor Lifetimes`_ for more details.
```

Comment:
Can you use an implicit reference instead? https://www.sphinx-doc.org/en/1.7/markup/inline.html#cross-referencing-arbitrary-locations